bridge/client/app/_interfaces/git-upstream.ts

```diff
@@ -0,0 +1,51 @@
+export interface IGitData {
+  gitRemoteURL?: string;
+  gitUser?: string;
+  gitToken?: string;
+  gitFormValid?: boolean;
+}
+
+export interface IRequiredGitData extends IGitData {
+  gitRemoteURL: string;
+  gitUser?: string;
```

Comment:
I guess re-defining this property is not necessary as it is already marked as optional in `IGitData`